ibft/storage/ibft_store.go

```diff
@@ -34,7 +34,7 @@ var (
 	metricsHighestDecided = promauto.NewGaugeVec(prometheus.GaugeOpts{
 		Name: "ssv:validator:ibft_highest_decided",
 		Help: "The highest decided sequence number",
-	}, []string{"lambda", "pubKey"})
+	}, []string{"identifier", "pubKey"})
```

Comment:
need to align Grafana as well

Reply:
@nivBlox Could you please share the project link where I need to change it as well?

Reply:
@nivBlox thanks, done